packages/twenty-front/src/modules/settings/accounts/components/SettingAccountsBlocklistContainer.tsx

```diff
@@ -0,0 +1,25 @@
+import { SettingsAccountsBlocklistContactRow } from '@/settings/accounts/components/SettingAccountBlocklistContactRow';
+import { BlocklistContext } from '@/settings/accounts/contexts/BlocklistContext';
+import styled from '@emotion/styled';
+import { useContext } from 'react';
+
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.spacing(2)};
+`;
+
+export const SettingAccountsBlocklistContainer = () => {
+  const { blocklist } = useContext(BlocklistContext);
+  return (
+    <StyledContainer>
+      <SettingsAccountsBlocklistContactRow />
```

Comment:
I'm surprise we have this component without any prop

Reply:
For an empty row, the state is empty, I am filling, the `item` prop is optional